app/controllers/v0/sessions_controller.rb

```diff
@@ -81,9 +81,7 @@ def saml_user
     end
 
     def create_saml_user
-      user = User.new(user_attributes)
-      user = Decorators::MviUserDecorator.new(user).create unless user.loa1? || user.gender.nil?
```

Comment:
Is the special-case `if gender.nil? then dont try MVI` logic still alive somewhere?  It is theoretically possible for ID.me to return a `nil` gender.